src/views/Question.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="question-wrapper">
+    <div class="bg-img"><img src="../assets/other-3.png" /></div>
```

Comment:
Perhaps can try use `background-image` CSS for this since this image is decorational (not part of the content of the page).